packages/ndla-ui/src/Article/Article.tsx

```diff
@@ -102,13 +102,13 @@ const MSGboxWrapper = styled.div`
 const ArticleFavoritesButtonWrapper = styled.div`
   display: flex;
   justify-content: flex-end;
-  transform: translate(${spacingUnit * 0.25}px, -${spacing.normal});
+  transform: translate(${spacing.xsmall});
```

Comment:
Det var bare den første delen som skulle byttes ut, den andre må bli! Altså
`translate(${spacing.xsmall}, -${spacing.medium});`

Reply:
On it!